heft-plugins/heft-storybook-plugin/src/StorybookPlugin.ts

```diff
@@ -117,14 +127,25 @@ export class StorybookPlugin implements IHeftPlugin<IStorybookPluginOptions> {
         '(EXPERIMENTAL) Used by the "@rushstack/heft-storybook-plugin" package to launch Storybook.'
     });
 
+    const storybookStaticBuildParameters: IHeftFlagParameter = heftSession.commandLine.registerFlagParameter({
+      associatedActionNames: ['build'],
+      parameterLongName: '--storybook',
+      description:
+        '(EXPERIMENTAL) Used by the "@rushstack/heft-storybook-plugin" package to start static build Storybook.'
+    });
+
     heftSession.hooks.build.tap(PLUGIN_NAME, (build: IBuildStageContext) => {
-      if (!storybookParameters.actionAssociated || !storybookParameters.value) {
+      if ((!storybookParameters.actionAssociated || !storybookParameters.value)
+        && (!storybookStaticBuildParameters.actionAssociated || !storybookStaticBuildParameters.value))
+      {
         this._logger.terminal.writeVerboseLine(
           'The command line does not include "--storybook", so bundling will proceed without Storybook'
         );
         return;
       }
 
+      this._modulePath = storybookStaticBuildParameters.actionAssociated ? options.staticBuildModulePath: options.startupModulePath;
+
       this._logger.terminal.writeVerboseLine(
         'The command line includes "--storybook", redirecting Webpack to Storybook'
       );
```

Comment:
The webpack bundle is still disabled when storybook is running in non-start mode. I get the rationale behind it when in start mode (since you'd have a different start mode experience that you're targeting), but is it expected to be skipped when building the static assets?

Reply:
Make sense, updated